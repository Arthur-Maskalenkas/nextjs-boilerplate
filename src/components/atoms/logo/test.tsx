import React from 'react'
import { render, RenderResult } from '@testing-library/react'

import { Logo } from '.'

type SutTypes = {
	sut: RenderResult
}

const makeSut = (): SutTypes => {
  const sut = render(<Logo />)

  return {
    sut
  }
}

describe('Logo', () => {
  test('Should render Logo correctly', () => {
    const { sut: { container } } = makeSut()

    expect(container).toMatchSnapshot()
  })
})
