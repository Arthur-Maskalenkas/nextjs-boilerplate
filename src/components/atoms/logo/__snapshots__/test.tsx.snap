// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Logo Should render Logo correctly 1`] = `
<div>
  <a
    aria-label="Logo"
    class="container"
    href="/#"
  >
    Arthur
  </a>
</div>
`;
